src/Propel/Generator/Builder/Om/AbstractOMBuilder.php

```diff
@@ -992,12 +992,17 @@ public function getBehaviorContentBase($contentName, $modifier)
      *
      * @param  string $filename
      * @param  array  $vars
-     * @param  string $templateDir
+     * @param  string|null $templateDir
      * @return string
      */
-    public function renderTemplate($filename, $vars = [], $templateDir = '/templates/')
+    public function renderTemplate($filename, $vars = [], $templateDir = null)
     {
-        $filePath = __DIR__ . $templateDir . $filename;
+        if (!$templateDir) {
+            $filePath = __DIR__ . '/templates/' . $filename;
+        } else {
+            $filePath = $templateDir . $filename;
```

Comment:
I really like the style used in https://github.com/propelorm/Propel2/pull/1491/files where the code uses `realpath` which would be less error-prone. Can we do something like that in this method?